docs/source/getting_started/authoring_your_workflow.md

```diff
@@ -140,47 +133,46 @@ def my_task(
 
 ## 4. Manage installation for third-party dependencies
 
-Latch uses Dockerfiles for dependencies management, which allow you to define the computing environment that your task will execute in.
+Under the hood, Latch uses Dockerfiles for dependency management, which allows a user to define the computing environment that their task will execute in. In most cases, there is no need for a user to write a Dockerfile, as the Latch SDK will automatically generate one.
 
-There's no need to create your Dockerfile from scratch as one is already provided when you initialized your workflow.
+We need `samtools` and `autoconf`. We can install these dependencies by adding them to the `system-requirements.txt` file, as shown [here](../basics/defining_environment.md#system-debian-packages). We will create and add the following lines to the file:
```

Comment:
grug need samtools. grug can install dependency by adding to system-requirements. grug will create and add the following lines

Reply:
```suggestion
Install `samtools` and `autoconf` by [adding them to the dependency list in `system-requirements.txt`:](../basics/defining_environment.md#system-debian-packages)
```